src/screens/sharing/Sharing.tsx

```diff
@@ -34,7 +34,13 @@ export const SharingScreen = () => {
         [platform]: () => {
           switch (platform) {
             case 'instagram': {
-              shareInstagramStory(theme.colors.mainBackground);
+              shareInstagramStory(
+                theme.colors.mainBackground,
+                Platform.select({
+                  android: i18n.translate('Sharing.Message'),
```

Comment:
Due to limitation of Instagram API on Android, it cannot share as a story (which is url). So, we use message instead.